pr_testing/run-pr-unittests.sh

```diff
@@ -10,57 +10,58 @@ cd $WORKSPACE/${CMSSW_VERSION}
 CMSSW_PKG_COUNT=$(ls -d $LOCALRT/src/*/* | wc -l)
 REPORT_OPTS="--report-url ${PR_RESULT_URL} $NO_POST"
 
-rm -f ${RESULTS_DIR}/unittestGPU.txt
-mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+rm -f ${RESULTS_DIR}/unittest${GPU_FLAVOR}.txt
+mark_commit_status_all_prs "unittests/${GPU_FLAVOR}" 'pending' -u "${BUILD_URL}" -d "Running tests" || true
```

Comment:
I hope `GPU_FLAVOR` is lower case in this script ..... right?

Reply:
yes, see e.g. https://cmssdt.cern.ch/jenkins/job/ib-run-pr-unittests/833/parameters/